exercises/robot/manipulation_station_io.ipynb

```diff
@@ -0,0 +1,535 @@
+    "    !pip install pyngrok  # Install pyngrok.\n",
+    "    server_args = ['--ngrok_http_tunnel']\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "P8HNbsnwSkzE"
+   },
+   "outputs": [],
+   "source": [
+    "# Start a single meshcat server instance to use for the remainder of this notebook.\n",
+    "from meshcat.servers.zmqserver import start_zmq_server_as_subprocess\n",
+    "proc, zmq_url, web_url = start_zmq_server_as_subprocess(server_args=server_args)\n",
+    "\n",
+    "import numpy as np\n",
+    "from functools import partial\n",
+    "from IPython.display import display\n",
+    "from ipywidgets import FloatSlider, Layout\n",
+    "from pydrake.all import JointIndex"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "bClJCCHTKBb5"
+   },
+   "source": [
+    "## Access System Input/Output Values\n",
+    "In this exercise, you will explore the [ManipulationStation](https://drake.mit.edu/pydrake/pydrake.examples.manipulation_station.html?highlight=manipulationstation#pydrake.examples.manipulation_station.ManipulationStation) that was mentioned during the lecture. The system diagram is shown below. \n",
+    "<img src=\"https://raw.githubusercontent.com/RussTedrake/manipulation/master/figures/exercises/manipulation_station_diagram.png\" width=\"1000\">\n",
+    "\n",
+    "You should recall that the orange ports are the ones that do not exist for the actual hardware platform.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "cR5vYf8Box0L"
+   },
+   "source": [
+    "Now we construct a ManipulationStation object and finalize the system setting. To get a sense of what this manipulation station looks like, you can open the meshcat viewer from the generated link as usual. There should be a bookshelf and a Kuka arm with a gripper attached (it might take a few seconds to load)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "SN1HkYCy8VJo"
+   },
+   "outputs": [],
+   "source": [
+    "from pydrake.examples.manipulation_station import ManipulationStation\n",
+    "from pydrake.systems.framework import DiagramBuilder\n",
+    "from pydrake.systems.meshcat_visualizer import MeshcatVisualizer\n",
+    "from pydrake.systems.analysis import Simulator\n",
+    "\n",
+    "station = ManipulationStation()\n",
+    "station.SetupManipulationClassStation()\n",
+    "plant = station.get_mutable_multibody_plant()\n",
+    "station.Finalize()\n",
+    "\n",
+    "builder = DiagramBuilder()\n",
+    "builder.AddSystem(station)\n",
+    "scene_graph = station.get_mutable_scene_graph()\n",
+    "meshcat = MeshcatVisualizer(scene_graph,\n",
+    "            zmq_url=zmq_url,\n",
+    "            frames_opacity=0.8)\n",
```

Comment:
why frames_opacity?  you're not drawing any frames?

Reply:
Got rid of this line!